src/main/java/Duke.java

```diff
@@ -10,16 +10,70 @@ public static void main(String[] args) {
         Scanner sc = new Scanner(System.in);
         String input;
         input = sc.nextLine();
-        List<String> list = new ArrayList<>();
+        List<Task> taskList = new ArrayList<>();
```

Comment:
Maybe can use tasks here instead of taskList?
```suggestion
        List<Task> tasks = new ArrayList<>();
```